piker/data/_ahab.py

```diff
@@ -143,27 +143,37 @@ async def process_logs_until(
                 if not entry:
                     continue
 
+                entry = entry.strip()
                 try:
-                    record = json.loads(entry.strip())
+                    record = json.loads(entry)
+
+                    if 'msg' in record:
+                        msg = record['msg']
+                    elif 'message' in record:
+                        msg = record['message']
+                    else:
+                        raise KeyError('Unexpected log format')
+
+                    level = record['level']
+
                 except json.JSONDecodeError:
-                    if 'Error' in entry:
-                        raise RuntimeError(entry)
-                    raise
+                    # if 'Error' in entry:
```

Comment:
Yeah this was obviously oriented around marketstore, so if it's not generic enough we should just drop it.